packages/plugin-core/src/commands/GoToSiblingCommand.ts

```diff
@@ -33,65 +34,207 @@ export class GoToSiblingCommand extends BasicCommand<
   }
 
   async execute(opts: CommandOpts) {
+    // Context for error messages
     const ctx = "GoToSiblingCommand";
-    const maybeTextEditor = VSCodeUtils.getActiveTextEditor();
-    if (!maybeTextEditor) {
+    // Get text editor
+    const textEditor = VSCodeUtils.getActiveTextEditor();
+    if (!textEditor) {
       window.showErrorMessage("You need to be in a note to use this command");
       return {
         msg: "no_editor" as const,
       };
     }
-    let value = "";
-    value = path.basename(maybeTextEditor.document.uri.fsPath, ".md");
-    let respNodes: NoteProps[] = [];
+    // Get fname of the active note
+    const fname = path.basename(textEditor.document.uri.fsPath, ".md");
+    // Get workspace
+    const workspace = ExtensionProvider.getDWorkspace();
+    // Get the active note
+    const note = await this.getActiveNote(workspace.engine, fname);
```

Comment:
you can use [getActiveNote](https://github.com/dendronhq/dendron/blob/enchance/wrap-month-journal-nav/packages/plugin-core/src/WSUtilsV2.ts#L226:L226)